common/beerocks/hostapd/include/configuration.h

```diff
@@ -97,10 +97,33 @@ class Configuration {
      * @brief for debug: return the last internal message
      * @details each action on this class changes its internal
      * message (similar to errno) - for debug usage
-     * @return s string describing the last message
+     * @return string describing the last message
      */
     const std::string &get_last_message() const;
 
+private:
+    /**
+     * @brief helper: get exiting vap
+     * @details any function that works on a specific vap does
+     * the same: search the vap, set a variable when found
+     * and set an error state when not found, this function
+     * does it instead of copy/paste the same code
+     * @param the calling function, string (to report error)
+     * @param the requested vap, string
+     * @return tuple<bool,std::vector<std::string>*> (found/not found, a 
+     * pointer to the vap's array)
+     */
+    std::tuple<bool, std::vector<std::string> *> get_vap(const std::string &calling_function,
```

Comment:
There's no need to return a tuple since the pointer takes the role of the boolean.